industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/tyk2/PREPARATION_DETAILS.md

```diff
@@ -0,0 +1,6 @@
+# Preparation done in Schrodinger suite 2024-2 (patch 1)
+# import tyk2_protein.pdb into maestro.  Contains an UNK residue which turns out to be NME.   Remove cap residues
+# to allow the prepwizard capping process to work on it. Just used delete atoms, it put them back in the same place anyway.
+# Run prepwizard
+# Changed residue type of NMA to NME and number to 1178 in the gui
+# all done for tyk2
```

Comment:
Could you please add a bullet point describing the process for preparing the ligands (i.e. removing the neutral forms of the ligands where two protonation states were present). Thanks!